tests/dashbio_demos/app_needle_plot.py

```diff
@@ -103,108 +103,258 @@ def description():
-                                                    {
-                                                        'label': 'Upload dataset',
-                                                        'value': FILE_KEY
-                                                    },
-                                                    {
-                                                        'label': 'UniProt dataset',
-                                                        'value': DATABASE_KEY
-                                                    },
-                                                ],
-                                                value=DEMO_KEY
-                                            )
+                                        dcc.Input(
+                                            id='needle-sequence-input',
+                                            value='',
+                                            type='text',
+                                            placeholder='TP53, DDX3X, SMARCA4, ...',
+                                        ),
+                                        html.Button(
+                                            id='needle-search-sequence-button',
+                                            children='submit',
+                                            n_clicks=0,
+                                            n_clicks_timestamp=0,
+                                        )
+                                    ]
+                                ),
+                                html.Div(
+                                    id='needle-uniprot-div',
+                                )
+                            ]
+                        ),
+
+                        html.Div(
+                            id='needle-%s-div' % FILE_KEY,
+                            className='needle-load-option-div',
+                            children=[
+                                html.Div(
+                                    id='needle-mutdata-file-div',
+                                    title='Mutation data files are json files containing '
+                                          'the fields :\n'
+                                          '- "x" (protein coordinate of the mutation) \n'
+                                          '- "y" (number of recorded mutations) \n'
+                                          '- "mutationGroups" (type of mutations) \n'
+                                          '- "domains" (protein domains.\n '
+                                          '"x", "y", "mutationGroups" are arrays, they '
+                                          'must have the same length or be empty, '
+                                          '"x" is required. The "domains" is an array of '
+                                          'json objects with the required fields "name" '
+                                          'and "coord", which are a string and a string '
```

Comment:
This is very unclear... Are both "name" and "coord" strings?

Reply:
@Bachibouzouk can answer this one :)

Reply:
Yes they are both strings

Reply:
If we talk about two fields ("name" and "coord") but then give just one example, there's no way to guess if it's an example value for "name" or for "coord"?!?
Continuing the conversation over at #277.